pkg/sdk/go/policies.go

```diff
@@ -21,6 +21,7 @@ type Policy struct {
 	Subject   string    `json:"subject"`
 	Object    string    `json:"object"`
 	Actions   []string  `json:"actions"`
+	External  bool      `json:"external,omitempty"`
```

Comment:
There probably should be a comment explaining what this `External` means, as here it looks like it means something like "external policy"